data.json

```diff
@@ -0,0 +1 @@
+[{"model": "character_generator.movement", "pk": 1, "fields": {"name": "Walker", "base": 2, "full": 8}}, {"model": "character_generator.movement", "pk": 2, "fields": {"name": "Winged", "base": 8, "full": 40}}, {"model": "character_generator.movement", "pk": 3, "fields": {"name": "Walker", "base": 4, "full": 20}}, {"model": "character_generator.morph", "pk": 1, "fields": {"name": "Neo-Avian", "desc": "Neo-avians include ravens, crows, and gray parrots uplifted to human-level intelligence. Their physical sizes are much larger than their non-uplifted cousins (to the size of a human child), with larger heads for their increased brain size. Numerous transgenic modifications have been made to their wings, allowing them to retain limited flight capabilities at 1 g, but giving them a more bat-like physiology so they can bend and fold better and adding primitive digits for basic tool manipulation. Their toes are also more articulated and now accompanied with an opposable thumb. Neo-avians have adpated well to microgravity environments, and are favored for their small size and reduced resource use.", "mtype": "Common", "cost": 0, "avail": 50, "wt": 5, "dur": 25, "dr": 38, "ins": 2, "mox": 1, "vig": 0, "flex": 0, "movtype": [1, 2]}}, {"model": "character_generator.morph", "pk": 2, "fields": {"name": "Novacrab", "desc": "Bioengineered from coconut crab and spider crab stock and grown to a larger (human) size, novacrabs are ideal for hazardous work environments as well as vacworker, police, or bodyguard duties. They climb and handle microgravity well and can withstand a wide range of atmospheric pressure (and sudden pressure changes) from vacuum to deep sea. They are equipped with ten 2-meter long legs, massive claws, chitinous armor, compound eyes (with human -equivalent image resolution), gills, dexterous manipulatory digits on their fifth set of limbs, and transgenic vocal cords.", "mtype": "Pod", "cost": 4, "avail": 50, "wt": 9, "dur": 45, "dr": 68, "ins": 0, "mox": 0, "vig": 3, "flex": 0, "movtype": [3]}}, {"model": "character_generator.morph", "pk": 3, "fields": {"name": "Exalt", "desc": "Exalts are the common, non-specialized \"enhanced human\" model. They are genetically upgraded to make them healthier, smarter, and more attractive (based on ever-evolving beauty standards). A modified metabolism keeps them fit and athletic for the duration of an extended lifespan.", "mtype": "Common", "cost": 2, "avail": 70, "wt": 7, "dur": 35, "dr": 53, "ins": 1, "mox": 1, "vig": 1, "flex": 0, "movtype": [3]}}]
```

Comment:
cool I will load this in with manage.py next time I'll run the project, well done!